lib/capistrano/twingly.rb

```diff
@@ -3,6 +3,8 @@
 
 # Include default deployment tasks
 require "capistrano/deploy"
+require "capistrano/scm/git"
```

Comment:
not sure if we need a comment here, I think it's kind of self-explanatory what we're doing here... perhaps a newline like:
```suggestion

require "capistrano/scm/git"
```

@roback?

Reply:
I don't think we need a comment either, but a newline, sure

Reply:
f-pushed